.github/workflows/deploy.yml

```diff
@@ -22,50 +22,54 @@ jobs:
       Application: ${{ github.event.repository.name }}
     secrets: inherit
 
-  qa-via:
+  QA:
     needs: create-image
-    name: ${{ github.event.repository.name }}
-    uses: hypothesis/workflows/.github/workflows/eb-update.yml@main
+    uses: hypothesis/workflows/.github/workflows/deploy.yml@main
     with:
-      Application: ${{ github.event.repository.name }}
-      Environment: qa
-      Region: us-west-1
-      Operation: deploy
-      Version: latest
+      operation: deploy
+      github_environment_name: QA
+      github_environment_url: https://qa-via.hypothes.is/
+      aws_region: us-west-1
+      elasticbeanstalk_application: via
+      elasticbeanstalk_environment: qa
+      version: ${{ needs.create-image.outputs.docker_tag }}
     secrets: inherit
 
-  qa-lms-via:
+  QA (Edu):
```

Comment:
Not sure if the space and parens in the labels will work or not. I might need to change the job label to `QA_Edu` and add a `name: QA (Edu)` field below.